File: app/src/main/java/com/yu/functionbox/utils/ToastUtils.java
package com.yu.functionbox.utils;

import android.widget.Toast;

import com.yu.functionbox.FunctionBoxApplication;


/**
 * Created by yianzhang on 16/8/21.
 */
public class ToastUtils {

    public static void show(String s) {
        Toast.makeText(FunctionBoxApplication.getApplication(), s, Toast.LENGTH_SHORT).show();
    }
}

File: app/src/main/java/com/yu/functionbox/utils/ToastUtils.kt
package com.yu.functionbox.utils

import android.widget.Toast

import com.yu.functionbox.FunctionBoxApplication


/**
 * Object声明的对象实际为单例模式的对象
 */
object ToastUtils {

    fun show(s: String) {
        Toast.makeText(FunctionBoxApplication.application, s, Toast.LENGTH_SHORT).show()
    }
}
